views/posts/new.hbs

```diff
@@ -1,7 +1,9 @@
 <form id="new-post-form" action="/posts" method="post" enctype="multipart/form-data" >
   <label for="message">Message: </label>
   <input id="message" type="text" name="message" value="" />
-  <label for="image">Image: </label>
```

Comment:
Please, change this code back. We need  input attribute `name` for label, and class `post-img` on img tag required for tests.